main.go

```diff
@@ -551,3 +551,19 @@ func surveyName(stdin chan ecma48.Output) string {
 		}
 	}
 }
+
+// log2(36^16) = 82 bits of entropy should be enough
+const randIdentChars = "abcdefghijklmnopqrstuvwxyz0123456789"
+const randIdentLen = 16
```

Comment:
Hmmm, I think you could go with `base64url` characters (A-Z, a-z, 0-9, -. _), and reduce randIdentLen to 14 to fit in a few less characters :D

Reply:
Ah, note that some filesystems are not case-sensitive, which is why I avoided uppercase characters. Do you think I should add a comment explaining that?

Reply:
Oh! :sweat_smile:  Case-insensitive definitely makes sense! No need for the comment as an explanation!